backend/src/common/enums/models/models.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum Models {
```

Comment:
Do it such way https://github.com/BinaryStudioAcademy/bsa-2021-healthcare/pull/2/files#diff-ec3dfa7ba63d0684bbf43b6de2680eec8ed498e0acf8bda39475818f2b619cfb